package.json

```diff
@@ -28,7 +29,7 @@
     "test:watch": "npm run --silent test -- --watch",
     "_build:angular": "ng build clr-angular --configuration production",
     "_build:demo": "ng build clr-demo --configuration production",
-    "_build:storybook": "npm run docs:json && build-storybook --output-dir ./dist/docs",
+    "_build:storybook": "npm run docs:json && STORYBOOK_ANGULAR_PROJECT=clr-angular build-storybook --output-dir ./dist/docs",
```

Comment:
This change isn't needed since we are skipping the storybook build for Angular v14 and v15.